terraform.tf.j2

```diff
@@ -39,35 +39,75 @@ data "terraform_remote_state" "internal_compute" {
   }
 }
 
-data "terraform_remote_state" "metrics_infrastructure" {
+data "terraform_remote_state" "security-tools" {
   backend   = "s3"
   workspace = terraform.workspace
 
   config = {
     bucket         = "{{terraform.state_file_bucket}}"
-    key            = "terraform/dataworks/dataworks-metrics-infrastructure.tfstate"
+    key            = "terraform/dataworks/aws-security-tools.tfstate"
     region         = "{{terraform.state_file_region}}"
     encrypt        = true
     kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
     dynamodb_table = "remote_state_locks"
   }
 }
 
-data "terraform_remote_state" "security-tools" {
+data "terraform_remote_state" "artefact" {
   backend   = "s3"
+  workspace = "management"
+
+  config = {
+    bucket         = "{{terraform.state_file_bucket}}"
+    key            = "terraform/dataworks/management.tfstate"
+    region         = "{{terraform.state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+  }
+}
+
+data "terraform_remote_state" "aws_certificate_authority" {
+  backend = "s3"
   workspace = terraform.workspace
+  config = {
+    bucket         = "{{terraform.state_file_bucket}}"
+    key            = "terraform/dataworks/aws-certificate-authority.tfstate"
+    region         = "{{terraform.state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+  }
+}
 
+data "terraform_remote_state" "crypto" {
+  backend   = "s3"
+  workspace = local.crypto_workspace[local.management_account[local.environment]]
   config = {
     bucket         = "{{terraform.state_file_bucket}}"
-    key            = "terraform/dataworks/aws-security-tools.tfstate"
+    key            = "terraform/dataworks/aws-crypto.tfstate"
     region         = "{{terraform.state_file_region}}"
     encrypt        = true
     kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
     dynamodb_table = "remote_state_locks"
   }
 }
 
-data "terraform_remote_state" "artefact" {
+data "terraform_remote_state" "mgmt_ca" {
```

Comment:
can we stick to a naming convention - mgmt or management, not a mix of the two?

Reply:
I'm not adding these, just moving them in the file to be in the same location altogether so don't want to change them